app/platform/fabric/gateway/FabricGateway.ts

```diff
@@ -488,4 +488,18 @@ export class FabricGateway {
 
 		return null;
 	}
+
+	async getActiveOrderersList(channel_name) {
+		const network = await this.gateway.getNetwork(channel_name);
+		let orderers = [];
+		let ordererAtrributes;
+		for (let [key, value] of network.discoveryService.channel.committers) {
```

Comment:
[nitpick]: we can have meaningful variable name.

Reply:
@adityajoshi12,
Changes done as suggested.